_posts/zh/2019-03-19-bech32-sending-support.md

```diff
@@ -178,7 +178,8 @@ h2:not(:first-of-type) { margin-top: 3em; }
 [newsletter #42]: /zh/newsletters/2019/04/16/#bech32-发送支持
 [newsletter #43]: /zh/newsletters/2019/04/23/#bech32-发送支持
 [newsletter #44]: /zh/newsletters/2019/04/30/#bech32-发送支持
-[newsletter #45]: /en/newsletters/2019/05/07/#bech32-sending-support
+[newsletter #44]: /zh/newsletters/2019/04/30/#bech32-发送支持
```

Comment:
I think this line is a duplicate

Reply:
Indeed, that's correct! Thank you for checking. I tried it again locally, and this kind of link duplication cannot actually be detected by make production. This error occurred when I was performing git rebase -master. I apologize for the inconvenience; I'm still not very proficient with git operations at times.